src/blocks/cards.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+}
+
+.card__image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.card__text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  width: 166px;
+  height: 16px;
+  left: 17px;
+  top: 277px;
+  align-items: center;
+}
+
+.card__review {
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  height: 18px;
+  margin-bottom: 16px;
+}
+
+.card__price-amount {
+  color: #060b35;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__price-price {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__buy {
+  width: 166px;
+  height: 40px;
```

Comment:
> do not set width/height explicitly in such cases

same here

Reply:
the checklist, point 8
set vertical paddings and make the button spread all over the available space